cucumber/api/features/retrieve_api_key.feature

```diff
@@ -0,0 +1,8 @@
+Feature: Retrieving an API key with conjurctl
+
+  # We need to be in production environment to test this
```

Comment:
Can you explain why we need to be in production?

Reply:
because that's the environment our clients will be. When we are in dev env the error doesn't occur